certificate-service/app/src/main/java/se/inera/intyg/certificateservice/application/certificate/dto/GetCertificateInternalXmlResponse.java

```diff
@@ -15,6 +16,7 @@ public class GetCertificateInternalXmlResponse {
   String certificateType;
   String unitId;
   String xml;
```

Comment:
Something we missed in earlier PR with sent. Could you add CertificateRecipientDTO?